machine.go

```diff
@@ -652,7 +650,10 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 		prefix = "/usr"
 	}
 
-	// search for busybox; in some distros it's located under /sbin
+	/* Add additional binary locations to $PATH */
+	os.Setenv("PATH", os.Getenv("PATH")+":"+prefix+"/sbin")
```

Comment:
In a non-merged usr setup you want to have both paths added to the path, not just the one. On a merged-usr system they collapse into one thing which is fine. So i'd suggest simply dropping this change entirely as it doesn't have a benefit and has the potential to cause regressions